### PR TITLE
Align eyes deprecation

### PR DESCRIPTION
**Pull request by kvrooman**
Over the last year, the amount of options and toggles has been slowly increasing in FS. In order to manage this growth, I wanted to re-evaluate align_eyes. The face detection and face alignment models have been greatly improved with the additional of superior models and the retirement of older inaccurate ones. This largely removes the primary case for align_eyes which is to vertically align the pupils of the face due to inaccuracies in the alignment model and/or umeyama fitting routine.

In addition, this feature has also been rendered less useful by the fact that the training input face images are randomly rotated anyways prior to training. This rotation augmentation directly makes the align_eyes feature useless.  For this reason and in order to limit the option creep in the ui, I'd like to deprecate align_eyes.

I'm open to hearing any reasons or use cases that I have overlooked that call for this toggle's continuing inclusion. Thank you.

### CODE REVIEW

**Comment by torzdf** [comment]
I'm all for this, to be honest.

**Comment by torzdf** [comment]
One small bit of feedback.... It's annoying that we have to insert `pylint:disable=no-member` for cv2, but linter complains otherwise :(

My preference is to keep it on the line which triggers the linter. Putting it at the top of the function means that it will ignore any actual "no-member" errors that occur.

**Comment by kvrooman** [comment]
Not much feedback one way or another from other folks. Leave up for comments for two more days and then merge?

**Comment by torzdf** [comment]
I have no issue with that

**Comment by linnik** [comment]
Instead of completely removing this functionality, what do you think about moving it out to separate script?

**Comment by kvrooman** [comment]
Per feedback, I have added a tool to replicate the functionality. This simplifies the code-base but still allows a means of aligning a image dataset if absolutely desired.

File: tools/cli.py [hunk 1]
```diff
@@ -25,31 +25,36 @@ def get_argument_list(self):
...
         argument_list = list()
         argument_list.append({
             "opts": ("-j", "--job"),
             "action": Radio,
             "type": str,
-            "choices": ("draw", "extract", "extract-large", "manual", "merge",
-                        "missing-alignments", "missing-frames", "legacy", "leftover-faces",
-                        "multi-faces", "no-faces", "reformat", "remove-faces", "remove-frames",
-                        "rename", "sort-x", "sort-y", "spatial", "update-hashes"),
+            "choices": ("draw", "extract", "extract-large", "extract-aligneyes", "manual",
```
Comment (torzdf) [inline tools/cli.py]: Could you move this back to an extract option please?

2 reasons:
1) The number of job options is already inordinately large, so would rather avoid adding more options here (I may change extract-large to be an extract option too)
2) We just convert it to a bool anyway, which is what the option will come in as, so there is no significant additional coding to be honest.

Comment (kvrooman) [reply tools/cli.py]: I'm confused? Making it an extraction option defeats the whole purpose of deprecating it from the extract options and extract toolchain ?

User feedback was to add it to tools if were removing it from extract options, so I added it here.

Comment (torzdf) [reply tools/cli.py]: I mean in the align tools menu (it currently resides in there as an option)

Comment (torzdf) [reply tools/cli.py]: image

Comment (kvrooman) [reply tools/cli.py]: yup, realized.. will do

**Comment by torzdf** [comment]
And sorry. I conflicted your commit again :(

**Comment by kvrooman** [comment]
Merged conflicts. Re-ran some testing and looks good.

**Pull request merged**